### ISSUE TITLE
Custom single class training, when should I stop training?

### ISSUE THREAD

**Issue by JoltonADsouza** [akozd/tensorflow_yolov3#5]
The training loss went down from some ~5000 to ~40 in som ~20000 iterations. I am unable to track the training on the graph. When do you think should I stop the training, at what number(eg. training loss: 2 or 3 or 4)?

**Comment by SrikarNamburu** [comment]
Hi,
I am not able to train the custom data.
I want to use pre-trained weights from the coco dataset and train my new dataset with 15 classes.

Error:

Restoring from checkpoint failed. This is most likely due to a mismatch between the current graph and the graph from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Assign requires the shapes of both tensors to match. LHS shape= [60] rhs shape= [255]

The new dataset has 15 classes that's why 60 filters
